src/plugins/flexibleLayout/toolbarProvider.js

```diff
@@ -61,13 +61,13 @@ function ToolbarProvider(openmct) {
         options: [
           {
             value: true,
-            icon: 'icon-columns',
-            title: 'Columns layout'
+            icon: 'icon-rows',
+            title: 'Switch to rows layout'
           },
           {
             value: false,
-            icon: 'icon-rows',
-            title: 'Rows layout'
+            icon: 'icon-columns',
+            title: 'Switch to columns layout'
```

Comment:
We need to make sure we update any e2e tests that use these titles / labels as locators

Reply:
Updated one test file.